### PR TITLE
Make volume clearer

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Renames `Volume` to `Payment Volume`. Not the tooltip part of the ticket here, since that's happening in a separate PR

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Make the "Volume" label clearer](https://trello.com/c/EMZlTUgt/516-make-the-volume-label-clearer)

File: app/client/components/pages/dashboardPage.jsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ class DashboardPage extends React.Component {
                 defaultTimeSeries="all_payments_volume"
                 filterObject="credit_transfer"
                 timeSeriesNameLabels={[
-                  ["all_payments_volume", "Volume"],
+                  ["all_payments_volume", "Payment Volume"],
```
Comment (tristanhcole) [inline app/client/components/pages/dashboardPage.jsx:152]: Does #429 change this? Does Volume mean all transactions on the system or just internal payments? If only internal payments then I think payment volume is better.

Comment (michielderoos) [reply app/client/components/pages/dashboardPage.jsx:152]: @tristanhcole It is all payments! What verbiage would you suggest to use here?

Comment (enjeyw) [reply app/client/components/pages/dashboardPage.jsx:152]: the tooltips _may_ help with this. Part of the reason this one is hard is that `Volume` on its own is concise and correct.

**Comment by tristanhcole** [comment]
I think we should close this for the meantime as (1) I don't think it necessarily makes it clearer and (2) I think we need [consistent labelling of transfers](https://trello.com/c/VHsuCaQ1/517-consistent-labeling-of-transfer-types-across-platform).

**Pull request closed without merging**